mappings/net/minecraft/util/math/intprovider/IntProvider.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_6017 net/minecraft/util/math/intprovider/IntProvider
+	FIELD field_29945 INT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_29946 VALUE_CODEC Lcom/mojang/serialization/Codec;
```

Comment:
This one should be `CODEC`, or `REGISTRY_CODEC` (that kind of codec that dispatches from registry for a type field).
The other one is an impl detail.

Reply:
I named IntProviders literally like `FloatProviders`, so they need a change as well I guess.